src/registry.jl

```diff
@@ -143,3 +143,37 @@ add_saver(format"PLY_BINARY", :MeshIO)
 add_format(format"2DM", "MESH2D\n", ".2dm")
 add_loader(format"2DM", :MeshIO)
 add_saver(format"2DM", :MeshIO)
+
+
+add_format(format"OFF", "OFF\n", ".off")
+add_loader(format"OFF", :MeshIO)
+add_saver(format"OFF", :MeshIO)
+
+function detect_stlbinary(io)
+    size_header = 80+sizeof(Uint32)
+    size_triangleblock = (4*3*sizeof(Float32)) + sizeof(Uint16)
+
+    position(io) != 0 && return false
+    seekend(io)
+    len = position(io)
+    seekstart(io)
+    len < size_header && return false
+    
+    header = readbytes(io, 80)
+    header[1:6] == b"solid " && return false
+    number_of_triangle_blocks = read(io, Uint32)
+     #1 normal, 3 vertices in Float32 + attrib count, usually 0
+    len != (number_of_triangle_blocks*size_triangleblock)+size_header && return false
```

Comment:
Why oh why do people design file formats with magic numbers?? This looks to be as careful as you can be.